src/wolframite/base/package.clj

```diff
@@ -0,0 +1,49 @@
+(ns wolframite.base.package
+  "The place to be for loading and manipulating Wolfram packages.
+"
+  (:require
+   [clojure.string :as string]
```

Comment:
```suggestion
   [clojure.string :as str]
```
Str is the alias used by convention

Reply:
I don't like it but I should accept it for this library.

Reply:
Conventions are rarely uniformly liked :)